### DISCUSSION TITLE
What is the purpose of the git2 binary?

### DISCUSSION THREAD

**Discussion by scottmckendry** [libgit2/libgit2#6805] [Q&A] [answered]
I'm packaging the libgit2 library for Windows via Chocolatey and moderators have asked about the git2.exe generated by CMake. 
I've searched for docs and other discussions where this is clarified but have been unable to find anything. Can anyone give me a quick summary or point me in the right direction?

**Comment by ethomson** [comment, marked as answer]
It's a command-line interface that is primarily meant to aide development and debugging. Here's some more information: https://github.com/libgit2/libgit2/pull/6133

If it were _me_, I would not package it with libgit2. I _might_ package it as its own separate application but there's little demand for such a thing from end-users.

**Reply by scottmckendry** [reply]
Thanks so much!
